index.js

```diff
@@ -9,7 +9,13 @@ const globals = require('./constants/globals.json');
 const xmlBuilder = new xml.Builder();
 xmlBuilder.options.rootName = 'request';
 
-let config;
+let config = {
```

Comment:
Is this default config meant to just have a test environment? So if the user never calls .init() or override with their own configuration then it will use test credentials?